Components/alert.css

```diff
@@ -1,40 +1,37 @@
 
 .components-alert-display{
-    border: 1px solid var(--color-black);
+    border: 1px solid var(--color-gray-200);
     padding: 1rem;
     width: 60%;
   }
 .alert-box{
     padding: 1rem;
-    border: 1px solid var(--color-black);
+    border: 1px solid var(--color-gray-600);
     margin: 0.5rem;
     border-radius:1rem;
     width: 60%;
     height: auto;
     display: flex;
     justify-content: center;
```

Comment:
you can use flex properties as helper class

Reply:
Great thanks, done the required changes, created a helper class for flex properties.